src/app/storage/duck/operations.ts

```diff
@@ -66,6 +67,42 @@ const addStorage = storageValues => {
   };
 };
 
+const updateStorage = storageValues => {
+  return async (dispatch, getState) => {
+    try {
+      const state = getState();
+      const { migMeta } = state;
+      const client: IClusterClient = ClientFactory.hostCluster(getState());
+
+      const tokenSecret = updateStorageSecret(
+        storageValues.secret,
+        storageValues.accessKey
+      );
+
+      const migStorage = updateMigStorage(
+        storageValues.bucketName,
+        storageValues.bucketRegion,
+      );
+
+      const secretResource = new CoreNamespacedResource(
+        CoreNamespacedResourceKind.Secret,
+        migMeta.namespace
+      );
+      const migStorageResource = new MigResource(MigResourceKind.MigStorage, migMeta.namespace);
+
+      const arr = await Promise.all([
+        client.patch(secretResource, storageValues.name, tokenSecret),
+        client.patch(migStorageResource, storageValues.name, migStorage),
+      ]);
+
+      dispatch(fetchStorage());
```

Comment:
Wasn't sure if we are supposed to issue an updatedSuccess to redux...could use some help knowing what the proper thing to do is around here.

Reply:
Yeah we have been sending success actions at the end of successful operations/api calls. They normally have just changed loading status & we havent implemented a success alert system.  We should open an issue to add  success notifications that auto dismiss after a set time.

Reply:
I see @andyvesel wrote up an issue to capture the need for success alert system:  https://github.com/fusor/mig-ui/issues/214

Reply:
Addressed.